src/main/java/com/emc/mongoose/storage/driver/kafka/KafkaStorageDriver.java

```diff
@@ -159,13 +166,27 @@ protected String requestNewAuthToken(Credential credential) {
       final I lastPrevItem,
       final int count)
       throws IOException {
-
+    
     if (listWasCalled) {
       throw new EOFException();
     }
 
     val buff = new ArrayList<I>(1);
-    buff.add(itemFactory.getItem(path + prefix, 0, 0));
+
+    if (itemFactory instanceof PathItemFactoryImpl) {
+      Properties properties = new Properties();
```

Comment:
use type inference and final modifier where possible
`val properties = new Properties();`
or
`final var properties = new Properties();`